pandas/tests/indexing/test_scalar.py

```diff
@@ -103,24 +76,25 @@ def test_at_iat_coercion(self):
         xp = s.values[5]
         assert result == xp
 
+    @pytest.mark.parametrize("idx", [lambda ser: ser.iat[1], lambda ser: ser.iloc[1]])
```

Comment:
there's a fixture indexer_al for at/loc, could make a similar one for iat/iloc

Reply:
Sure. Made a fixture